keyboards/atreus62/keymaps/hvp/config.h

```diff
@@ -0,0 +1,3 @@
+#define TAPPING_TERM 150
```

Comment:
Ideally, this should have an include guard, to ensure no weird issues. 

```suggestion
#pragma once 

#define TAPPING_TERM 150
```

Reply:
Fixed